lore/settings.py

```diff
@@ -353,5 +353,16 @@ def get_var(name, default):
     }
 }
 HAYSTACK_SIGNAL_PROCESSOR = 'haystack.signals.RealtimeSignalProcessor'
-ALLOW_CACHING = get_var("ALLOW_CACHING", get_var("ALLOW_CACHING", False))
+
 XANALYTICS_URL = get_var('XANALYTICS_URL', "")
+
+CACHES = {
+    "default": {
+        "BACKEND": get_var(
+            "CACHE_BACKEND", "django.core.cache.backends.locmem.LocMemCache")
+    },
+    "indexing": {
+        "BACKEND": "django.core.cache.backends.locmem.LocMemCache",
+        "TIMEOUT": get_var("CACHE_TIMEOUT", "60"),
```

Comment:
I misunderstood your comment, but it makes sense to update `indexing` to be more specific also. I'll change both.